src/cljs/analysis_viewer/subs.cljs

```diff
@@ -126,21 +126,24 @@
       data
       (utils/filter-map-vals data attr-filter))))
 
+(defn colored-and-filtered-data [data params filters]
+  (println "Total data objects " (count data))
```

Comment:
needed? Maybe use logging?

Reply:
I'm using the console as light logging in a couple of important places to make it easier to troubleshoot if something is weird.
What kind of logging do you suggest instead? I'm not using any lib since I don't think it is needed for the visualizer

Reply:
Good enough